ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Cleaner.java

```diff
@@ -189,6 +197,12 @@ private void clean(CompactionInfo ci, long minOpenTxnGLB, boolean metricsEnabled
           txnHandler.markCleaned(ci);
           return;
         }
+        if(MetaStoreUtils.isNoCleanUpSet(p.getParameters())){
+          // The table was marked no clean up true.
```

Comment:
Update the comment: "The partition is marked with no_cleanup=true"

Reply:
Done